assets/language/en.json

```diff
@@ -164,7 +164,7 @@
     "map_txt": "Map",
     "list_txt": "List",
     "report_of_the_day_txt": "Notification of the day ",
-    "location_txt": "Approximate location",
+    "location_txt": "Approximate location:",
```

Comment:
Remove `:`

Reply:
okey, I'll trust the provided translations. I was just confused that some languages have the : on this string while others don't

Reply:
You're right, some translations do contain ":". I will solve that on issue #80 ensuring that these characters are on the code side, not on the translation. Keep it without the ":" for this pull request,  thought

Reply:
perfect